client/my-sites/checkout/checkout-thank-you/licensing-thank-you-manual-activation-instructions.tsx

```diff
@@ -9,28 +9,127 @@ import ExternalLink from 'calypso/components/external-link';
 import LicensingActivation from 'calypso/components/jetpack/licensing-activation';
 import PageViewTracker from 'calypso/lib/analytics/page-view-tracker';
 import { addQueryArgs } from 'calypso/lib/url';
+import hasStandalonePlugin from 'calypso/my-sites/plans/jetpack-plans/has-standalone-plugin';
 import slugToSelectorProduct from 'calypso/my-sites/plans/jetpack-plans/slug-to-selector-product';
+import { SelectorProduct } from 'calypso/my-sites/plans/jetpack-plans/types';
 import { recordTracksEvent } from 'calypso/state/analytics/actions';
 
 interface Props {
 	productSlug: string | 'no_product';
 	receiptId: number;
 }
 
+interface JetpackStandaloneActivationInstructionsProps {
+	product: SelectorProduct;
+}
+
+const JetpackPluginActivationInstructions: FC = () => {
```

Comment:
Can we extract this component to its own file?

Reply:
I was hesitating earlier to move this in a separate file earlier but yeah If you agree it is good to have it in separate file then I agree 100%. 👍